db/db.go

```diff
@@ -114,18 +114,16 @@ func NewMinio(env e2e.Environment, name, bktName string, opts ...Option) *e2emon
 				"sh",
 				"-c",
 				command+fmt.Sprintf(
-					"su - me -s /bin/sh -c 'mkdir -p %s && %s /opt/bin/minio server --address :%v --console-address :%v --quiet %v'",
+					"su - me -s /bin/sh -c 'mkdir -p %s && %s /opt/bin/minio server --address :%v --quiet %v'",
```

Comment:
Is the default port name of the console 8080?

Reply:
Fixed in https://github.com/efficientgo/e2e/pull/61/commits/0aa4b760c2838345b461465dfd1d729c0a710c1d. 👍